test/test_loggers.py

```diff
@@ -98,6 +110,13 @@ def test_log_video(self, steps, tb_logger):
                 step=steps[i] if steps else None,
             )
 
+    def test_log_hparams(self, tb_logger):
+        config = get_example_config()
+        del config["nested"]  # not supported in tensorboard
+        del config["list"]  # not supported in tensorboard
+        del config["tuple"]  # not supported in tensorboard
```

Comment:
what is the error message? Is it explicit on their end? Otherwsie we may want to capture it and test it (ie test that we get a good error message whenever a nested dict is passed)

Reply:
it is quite good,
it will say that you tried to log a param of type {type} and that is not supported in {logger_name}